src/sparkname.cpp

```
@@ -327,8 +343,18 @@ bool CSparkNameManager::ValidateSparkNameData(const CSparkNameTxData &sparkNameD
     else if (sparkNameData.additionalInfo.size() > 1024)
         errorDescription = "additional info is too long";
 
-    else if (sparkNameData.sparkNameValidityBlocks > 365*24*24*10)
-        errorDescription = "transaction can't be valid for more than 10 years";
+    else if (sparkNameData.sparkNameValidityBlocks > 365*24*24*10) {
```

Cppcheck (2.19.0)

[error] Uninitialized variable

(uninitvar)